gcloud/test__helpers.py

```diff
@@ -148,6 +148,38 @@ def test_value_set(self):
             self.assertEqual(dataset_id, APP_ENGINE_ID)
 
 
+class Test__get_credentials_file_project_id(unittest2.TestCase):
+
+    def _callFUT(self):
+        from gcloud._helpers import _file_project_id
+        return _file_project_id()
+
+    def setUp(self):
+        import os
+
+        self.old_env = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', '')
+        os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = ''
+
+    def tearDown(self):
+        import os
+        os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = self.old_env
```

Comment:
If the key wasn't in `ow.environ` during `setUp`, then ensure that it isn't present (as opposed to present with value `None`) during `tearDown`.